### PR TITLE
Add Modern CSV 1.3.

### PR DESCRIPTION
**Pull request by evanemm**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [ x] `brew cask audit --download {{cask_file}}` is error-free.
- [ x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x ] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [ x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [ x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x ] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [ x] `brew cask install {{cask_file}}` worked successfully.
- [ x] `brew cask uninstall {{cask_file}}` worked successfully.
- [ x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x ] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/moderncsv.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask "moderncsv" do
```
Comment (vitorgalvao) [inline Casks/moderncsv.rb:1]: Does not follow [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md). Needs to be `modern-csv`.

File: Casks/moderncsv.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  version "1.3"
+  sha256 "3803503571e1e82f386ff5bef20972d9f99779f540e222c2b88328f86aa35f5f"
+
+  url "https://www.moderncsv.com/release/ModernCSV-Mac-v1.3.dmg"
```
Comment (vitorgalvao) [inline Casks/moderncsv.rb:5]:
```suggestion
  url "https://www.moderncsv.com/release/ModernCSV-Mac-v#{version}.dmg"
```

Comment (vitorgalvao) [reply Casks/moderncsv.rb:5]: Please add an [`appcast`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

File: Casks/moderncsv.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  name "Modern CSV"
+  desc "CSV editor"
+  homepage "https://www.moderncsv.com/"
+
+  app "Modern CSV.app"
```
Comment (vitorgalvao) [inline Casks/moderncsv.rb:10]: Please add a [`zap`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by evanemm** [comment]
My bad. Just fixed.

**Review by reitermarkus — APPROVED** [review]

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**